backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/AssignmentLinksApi.cs

```diff
@@ -103,18 +125,81 @@ public AssignmentLinksApi(ILogger<AssignmentLinksApi> logger, IMapper mapper)
         public async Task<IActionResult> DeleteLink(
             [HttpTrigger(AuthorizationLevel.Anonymous, "delete", Route = "assignments/{assignmentId}/links/{linkId}")] HttpRequest req,
             [Table(AssignmentLinksTableName)] CloudTable assignmentLinksTable,
-            [Table(AssignmentLinksTableName, "{assignmentId}", "{linkId}")] AssignmentLinkEntity entityToDelete)
+            [Table(AssignmentLinksTableName, "{assignmentId}", "{linkId}")] AssignmentLinkEntity entityToDelete,
+            [Assignment(AssignmentId = "{assignmentId}")] Assignment assignment,
+            [Platform] PlatformsClient platformsClient,
+            [LtiAdvantage] INrpsClient nrpsClient,
+            [Lti1] Lti1MembershipClient membershipClient)
         {
             if (entityToDelete == null)
+            {
+                _logger.LogInformation("Entity to delete is null");

```

Comment:
I am having following lines of thought here. Please see if any of these could apply here: - 
- I believe we are getting the entity via a storage interaction. Is it possible to get this information via App Insights? If yes, we could get rid of this log entry.
- The entry to delete turns out to be not present in the table. Should we consider returning `http204` here instead? That should help us get rid of the log entry without losing any information about what happened on the server.

Reply:
changed the response to `NoContentResult` and removed the logging